spec/features/pages/risk.rb

```diff
@@ -34,27 +35,23 @@ def fill_in_risk_from_others
       save_and_continue
     end
 
+    def fill_in_discrimination
+      fill_in_optional_details('Are they a risk to females?', @risk, :risk_to_females)
```

Comment:
📓 This might be a point for improvement, at the moment I see little advantage having to specify here the exact label if its being managed by the locales.

It seems the DOM would probably require some changes to help us find the elements more accurately without using the current approach.